### PR TITLE
Refer to IBM docs

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Closes https://github.com/Qiskit/qiskit-terra/issues/10594.

Screenshot 2023-08-21 at 7 30 42 AM

### CODE REVIEW

File: docs/conf.py
Comment (javabster) [inline docs/conf.py:133]:
```suggestion
    "announcement_url": "https://docs.quantum-computing.ibm.com/",
```

File: docs/conf.py
Comment (jakelishman) [inline docs/conf.py:134]: This feels like a slightly weird coupling between the theme and the repository, but not a huge deal.

Minor: the split of "namespacing" one variable with `theme_` but not the others is a bit odd to me.

(Personally, I'd prefer we namespace everything Qiskit-specific with a `qiskit_` prefix, since we have to share the namespace with everything Sphinx-related, and reading the conf file, it's unclear to me now what's related to the theme and what's not, but that's a relatively minor issue and out of scope any way.)

**Review by jakelishman — COMMENTED** [review]
A bit minor, but as I understand it, this is directly lifted from Abby's PR Qiskit/qiskit-metapackage#1791 - it might be nice to make sure Abby's credited at least as a co-author on the PR.

File: docs/conf.py
Comment (Eric-Arellano) [reply docs/conf.py:134]: Yeah I agree. Too late to change now that it's been released though.

**Review by jakelishman — APPROVED** [review]
As far as I'm concerned this is fine, but probably best to get a review from someone properly on the docs squad / involved in the 1XP discussions before merge.

**Comment by Eric-Arellano** [comment]
> As far as I'm concerned this is fine, but probably best to get a review from someone properly on the docs squad / involved in the 1XP discussions before merge.

We've internally shared screenshots with Jules and she approved :)

**Comment by Eric-Arellano** [comment]
Ah, @jakelishman @mtreinish I forgot to put stable-backport-potential when opening this. It would help the 1XP team a lot if this went out on stable docs, not only the dev docs.

**Comment by jakelishman** [comment]
Either way, best have Abby sign off on the PR to keep the full spirit of the review process, since she'd suggested changes.

**Comment by jakelishman** [comment]
And yeah, this is inherently stable for backport, so no trouble adding the label.

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**
